pkg/agent/openflow/pipeline_windows.go

```diff
@@ -311,3 +345,51 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 	}
 	return []binding.Flow{c.l3FwdFlowToRemoteViaGW(localGatewayMAC, *peerPodCIDR, category)}
 }
+
+// clusterIPHostNetworkFlows installs flows needed for a ClusterIP Service which is accessed from Windows host.
+func (c *client) clusterIPHostNetworkFlows(svcIP net.IP, gwIP net.IP, nodeIP net.IP, category cookie.Category) (flows []binding.Flow) {
+	flows = append(flows,
+		// Hostnetwork case needs SNAT so the SNAT mark is set.
```

Comment:
All the flows in the func expect the last one should be shared by all Services? Then we should add the common flows only once, but not add them again for each Service?

Reply:
You're right. Updated.